monitoring/metrics/frontend_metrics.py

```
@@ -0,0 +1,88 @@
+# Autonomica Frontend Metrics Configuration
+# This file configures Prometheus metrics collection for the frontend
+
+from prometheus_client import Counter, Histogram, Gauge, generate_latest
+from prometheus_client import CONTENT_TYPE_LATEST
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)